README.md

```diff
@@ -371,14 +373,18 @@ Or install it yourself as:
 
 Run them with:
 
-`rake`
+    rake
 
 Or to see warnings (try not to have any):
 
-`RUBYOPT=-w rake`
+    RUBYOPT=-w rake
 
 The tests are intentionally split into two test suites for reasons described in `Rakefile`.
 
+You can run an individual test using the [m](https://github.com/qrush/m) gem:
+
+    m spec/m spec/attr_extras/attr_initialize_spec.rb:48
+
```

Comment:
I added this "m" stuff on master (as a pair commit with you), so if you rebase this diff should be smaller.